angular.json

```diff
@@ -132,6 +132,7 @@
   "cli": {
     "schematicCollections": [
       "@angular-eslint/schematics"
-    ]
+    ],
+    "analytics": false
```

Comment:
Dlaczego dodałeś tę linię?

Reply:
Szczerze mowiac dlugo sam sie zastanawialem skad to się wzielo, i w koncu udalo mi sie znalezc odpowiedz. Jak uruchamiam server przez npm run start dostaje komunikat:
? Would you like to share pseudonymous usage data about this project with the Angular Team
at Google under Google's Privacy Policy at https://policies.google.com/privacy. For more
details and how to change this setting, see https://angular.io/analytics. (y/N)
Po wybraniu nie, w package pokazuje sie to ustawienie